packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss

```diff
@@ -1,33 +1,67 @@
 @use '@carbon/layout';
+@use '@carbon/colors';
 
-.container {
+.imageContainer {
   display: flex;
+  flex-direction: column;
   align-items: center;
 }
 
-.buttonCssReset {
-  max-width: layout.$spacing-10;
-  padding: 0;
-  margin: 0;
-  border: none;
-  background: none;
-  cursor: pointer;
+.placeholderText {
+  width: 57px;
+  height: 32px;
+  font-family: IBMPlexSans;
+  font-size: 12px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.33;
+  letter-spacing: 0.32px;
+  color: colors.$cool-gray-70;
+  width: 120px;
+  height: 120px;
+  padding: 44px 31px 44px 32px;
+  border: solid 1px colors.$cool-gray-80;
 }
 
-.placeholderIconContainer {
-  width: layout.$spacing-10;
-  height: layout.$spacing-10;
-  background-color: #d8d8d8;
+.editImage {
   display: flex;
+  flex-direction: row;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
+  width: 120px;
+  height: 48px;
```

Comment:
use `layout.$spacing-09`

Reply:
Thanks @usamaidrsk! We pushed some commits to with changes.